core/src/main/java/org/mskcc/cbio/portal/dao/DaoReferenceGenome.java

```diff
@@ -230,18 +230,17 @@ public static ReferenceGenome getReferenceGenomeByGenomeName(String genomeName)
         return byGenomeName.get(genomeName);
     }
     
-    
     /**
      * Retrieve reference genome of interest by genome name or genome assembly name
-     * @param buildName   Name of Reference Genome or Genome Assembly
+     * @param name   Name of Reference Genome or Genome Assembly
```

Comment:
perhaps add here that it returns the id for "GRCh37" if the name is not found.

Reply:
@pieterlukasse , the code already did that. Now, I am thinking about add a new param of species, since if the instance is for mouse, then by default we shall return the id for "GRCm37". what do you think?
```
            if (species.equalsIgnoreCase("mouse")) {
                return genomeInternalIds.get("GRCm37");
            } else if (species.equalsIgnoreCase("human")){
                return genomeInternalIds.get("GRCh37"); // NCBI_BUILD field was an optional in the past
            }  else {
                throw new DaoException("Species not supported yet");
            } 
```